Makefile

```diff
@@ -73,7 +73,7 @@ postsubmit-conformance:
 .PHONY: upload
 upload:
 	release/generate_crd.sh $(RELEASE_BRANCH) $(RELEASE)
-	release/s3_sync.sh $(RELEASE_BRANCH) $(RELEASE) $(ARTIFACT_BUCKET)
+	release/s3_sync.sh $(RELEASE_BRANCH) $(RELEASE) $(ARTIFACT_BUCKET) ""
```

Comment:
I think it works even if you don't pass in the quotes here. As far as I have seen, Bash automatically takes missing arguments to be empty strings.